hack/chains/kaniko-demo.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 source $(dirname $0)/_helpers.sh
-set -ue
```

Comment:
Why remove this?

Reply:
I think I did while debugging. Adding it back.